timesheet_automatic_remind/models/timesheet_employee_remind.py

```diff
@@ -0,0 +1,168 @@
+# -*- coding: utf-8 -*-
+# © 2015 Elico corp (www.elico-corp.com)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+import datetime
+import time
+import pytz
+from openerp import fields, models, api, _
+
+
+class TimesheetReport(models.TransientModel):
+    _name = "timesheet.employee.remind"
+    _description = "Timesheet Employee Remind Service"
+    _inherit = ['mail.thread']
+
+    MONDAY = 0
+    TUESDAY = 1
+    WEDNESDAY = 2
+    THURSDAY = 3
+    FRIDAY = 4
+    WEEKEND = [5, 6]
+
+    @api.model
+    def _start_remind(
+            self,
+            count_days=7,
+            workstart='9:00:00',
```

Comment:
![image](https://cloud.githubusercontent.com/assets/16707489/25221333/5833c14e-25e7-11e7-8cc1-138e3d6c8847.png)
those values could be modified in corn job view. They could be maintenanced easily in the future.